### ISSUE TITLE
ratings for destination_merge_outcome in destination_merge_outcome_ratings not initialized correctly (it is empty)

### ISSUE THREAD

**Issue by wellemut** [kartevonmorgen/FairSync#32]
Even though I choose the existing entry on kvm, the ratings-Ids somehow are not by default suggested in the destination document, but remain in the reference-document
![Image](https://github.com/user-attachments/assets/529f829e-3bb9-40fb-8c20-3dc08e2b7f82)

- [ ] Ratings should never get deleted by default.

**Comment by qknight** [comment]
new wisdom:

# create a new rating

POST  /v0/ratings

`{"entry":"a9177e600c4a4693969c636c1168860b","context":"diversity","title":"sehr gut","comment":"xxx 10 20 30 40 50 60","source":"g&b online","value":2}`


# get a rating by id

 GET /v0/ratings/3d1ac036bbbd4e718f6eece87b0d074b

# the struct rating
```
{
	"0": {
		"id": "3d1ac036bbbd4e718f6eece87b0d074b",
		"title": "sehr gut",
		"created": 1734529200,
		"value": 2,
		"context": "diversity",
		"comments": [
			{
				"id": "1270b39e47a9426b9724918b062ca3a1",
				"created": 1734529200,
				"text": "xxx 10 20 30 40 50 60"
			}
		],
		"source": "g&b online"
	}
}
```

**Comment by qknight** [comment]
Currently ratings are referenced by ID. In oder to archive an article and migrating data to a different one (merge) then we can either copy all the references & comments respectively or we can just add a reference from both.

**Comment by wellemut** [comment]
Thank you @qknight  and @flosse for the new interface: 

But how is it meant to be used? 
The strange behaviours is similar to the previous interface, so the problem remains: 
- You can see, I have chosen the existing entries as the destination and only that one has actually ratings. https://dev.kartevonmorgen.org/m/main/e/67bafaea8b0649caadb6b008a198c69d?c=49.8009%2C9.9813&z=16.00&mapColorMode=gray&search=%C3%B6koligenta
- But in the dialog, the ratings are shown with a checkbox only on the left, as if they would jsut exist in the import file. 
- So I have to check them, so they dont get lost.
- can we make them preselected, as all other existing data of the destination entry on the right side?
![grafik](https://github.com/user-attachments/assets/ca2c7a9f-ca22-40ed-a7d6-9b8a74d2ad75)

**Comment by qknight** [comment]
@wellemut this works similar to all the other merges. if you want them all from (1.) then you have to click (2.)

![grafik](https://github.com/user-attachments/assets/5018bf15-d20d-42e0-8ed7-bb311329c54f)

we don't automerge content in this merge dialog, we do this nowhere, we never did and never will.

**Comment by qknight** [comment]
i think this explains it all

**Comment by qknight** [comment]
i was wrong, there is a bug in the initialization of the target.

**Comment by qknight** [comment]
fixed with 63d09e84e691baa5713b2613ab28bf7a9c028fc7
